### PR TITLE
[Merged by Bors] - feat(CategoryTheory): a sequential limit of surjections is surjective

### PR DESCRIPTION
**Pull request by dagurtomas**
We prove that in a concrete category whose forgetful functor preserves sequential limits, a sequential limit of surjections is surjective. Since this will be applied to `LightProfinite`, we provide the necessary instance to be able to apply this lemma as well. 

---
- [x] depends on: #13319 

This will be used in a future PR to prove that a sequential limit of epimorphisms in `LightCondSet` is an epimorphism.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by joelriou** [comment]
Is not it possible to deduce this from the results in `CategoryTheory.CofilteredSystem`? (The API may have to be slightly developed though.)

**Comment by dagurtomas** [comment]
> Is not it possible to deduce this from the results in `CategoryTheory.CofilteredSystem`? (The API may have to be slightly developed though.)

That file deals mostly with cofiltered systems of *finite* sets. Here I have a cofiltered system (indexed by N^op) of light profinite sets, possibly inifinite. It is surely possible, however, to deduce this from a similar statement about inverse systems in `Type _`, because the forgetful functor from `LightProfinite` to `Type _` preserves the limits in question.

File: Mathlib/CategoryTheory/Limits/ConcreteCategory.lean
Comment (erdOne) [inline Mathlib/CategoryTheory/Limits/ConcreteCategory.lean:72]: I think the name given by the naming convention would be something like `surjective_π_app_zero_of_surjective_map`.

**Review by erdOne — APPROVED** [review]
LGTM modulo the nitpick on naming.

**Comment by dagurtomas** [comment]
Fixed the name

**Comment by erdOne** [comment]
Thanks!
maintainer merge

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**
